### DISCUSSION TITLE
Metrics port identifier for argo-events

### DISCUSSION THREAD

**Discussion by pcperera** [argoproj/argo-events#1979] [Q&A]
Hi all,

I'm trying to scrape argo-events through Victoria Metrics through the port identifier **metrics**. However the scrape not successful.  I see that only port-identifiers that are exposed through argo-events are **healthz** and **readyz**. 

How can I find the port where argo-events metrics are exposed ?

![image](https://user-images.githubusercontent.com/15628990/168969546-ea595ce7-aab5-433d-825a-4bb3d7e4e2cd.png)

Thanks
Priyanka

**Comment by whynowy** [comment]
7777

**Comment by whynowy** [comment]
It's in EventSource/Sensor pod spec.

**Comment by pcperera** [comment]
Thanks. Is the below podMetricsEndpoints configuration correct ?


![image](https://user-images.githubusercontent.com/15628990/169759414-8667d81a-613f-460d-aa67-71f3cfea4915.png)
